src/python/install.sh

```diff
@@ -700,6 +753,38 @@ export DEBIAN_FRONTEND=noninteractive
 
 REQUIRED_PKGS=""
 case ${ADJUSTED_ID} in
+    alpine)
+        REQUIRED_PKGS="${REQUIRED_PKGS} \
+            curl \
+            "
+
+        # ref. <https://github.com/docker-library/python/blob/2d4fb586c48b067b432cf56653ee2541d94fdd7d/3.11/alpine3.20/Dockerfile#L29>
+        REQUIRED_PKGS="${REQUIRED_PKGS} \
+            bluez-dev \
```

Comment:
Did you notice in testing that many of these packages were required?  IIRC the debian packages were included as they were necessary for compilation.  There's a few packages here (bluez-dev, a bluetooth lib) that I suspect are not required.  

Have you validated the build from source branch of this feature?  I'd be surprised if it works with only these changes.  If not, I think it's definitely OK to block installing from source on alpine.